package.json

```diff
@@ -20,20 +20,20 @@
     "react-test-renderer": ">=15"
   },
   "dependencies": {
-    "jest-diff": "test",
-    "jest-snapshot": "test",
-    "pretty-format": "^20.0.3",
+    "jest-diff": "^22.0.6",
+    "jest-snapshot": "^22.0.6",
+    "pretty-format": "^22.0.6",
     "strip-ansi": "^4.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-preset-env": "^1.6.0",
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react": "^6.24.1",
-    "eslint": "^4.3.0",
+    "eslint": "~4.13.0",
```

Comment:
I deleted the lockfile to update all deps within semver, and without also updating `eslint-config-callstack-io` we are hit by the babel-eslint bug.

Reply:
Sure, good move

Reply:
We can update the eslint config in the followup PR though